packages/renderer/src/App.svelte

```diff
@@ -132,7 +132,7 @@ window.events?.receive('display-troubleshooting', () => {
             type="container" />
         </Route>
         <!-- Same DeployPodToKube route, but instead we pass in the compose group name, then redirect to DeployPodToKube -->
-        <Route path="/compose/deploy-to-kube/:composeGroupName/:engineId/*" breadcrumb="Deploy to Kubernetes" let:meta>
+        <Route path="/compose-deploy-to-kube/:composeGroupName/:engineId/*" breadcrumb="Deploy to Kubernetes" let:meta>
```

Comment:
I would keep the `/compose` namespace to all the compose routing and use other differentiators instead of having separate routes at the beginning
-> change routing scheme after `/compose` rather than at the root level

if we have common prefix we can route or use components for the routing (I think @feloy wanted to introduce routing scheme/helpers at some point) but if we break at the root level, it'll be harder to introduce Routing components per scope

Reply:
Ok, second commit reverts the deploy-to-kube path and changes the details path instead. I just don't love this option because it makes the compose details different from image/volumes/pods/etc details instead, but we have to change one of them.

Reply:
we could change the route for image/volumes/pods/etc to be consistent 👍

Reply:
I found the related PR https://github.com/containers/podman-desktop/pull/4371

Reply:
I finally reverted it in this PR, as I didn't have enough perspective at this time. But it is visible in the first commit of the PR

Reply:
100% agreed on the solution, but I don't think we should hold up a minor fix to solve the broader problem. I've opened #4966 to track solving this based on what was in #4371.